packages/mask/src/components/InjectedComponents/ProfileCard/ProfileBar.tsx

```diff
@@ -147,14 +150,17 @@ export const ProfileBar = memo<ProfileBarProps>(
 
         return (
             <Box className={cx(classes.root, className)} {...rest} ref={containerRef}>
-                <div className={classes.avatar}>
-                    <img
+                <div className={classes.avatar} data-id={identity.identifier?.userId}>
```

Comment:
Where did you use the `data-id`? Just a visual helper?

Reply:
> Where did you use the `data-id`? Just a visual helper?

Yes, for debugging

```suggestion
                <div className={classes.avatar}>
```